scripts/exglobal_prep_emissions.py

```diff
@@ -19,8 +19,14 @@
     config = cast_strdict_as_dtypedict(os.environ)
 
     # Instantiate the emissions pre-processing task
-    # emissions = AerosolEmissions(config)
-    # emissions.initialize()
-    # emissions.configure()
-    # emissions.execute(emissions.task_config.DATA, emissions.task_config.APRUN)
-    # emissions.finalize()
+    emissions = ChemFireEmissions(config)
+    emissions.initialize()
+    emissions.configure()
+    emissions.execute()
+    emissions.finalize()
```

Comment:
Let's call these fire `fireemis`?

Reply:
You want to rename the object `emissions` here?

Reply:
yeah just because the other object is also emissions but from NEXUS, so better to be clearer I think